Add compatibility table (#311)
packit-service/ogr: COMPATIBILITY.md

# Compatibility table for services

Methods/properties not listed in here are implemented for all services.

> In case of an error, please report with issue.

Reviewer: ```suggestion
In case of an error, please [create a new issue](https://github.com/packit-service/ogr/issues/new).
```

## Comments

### `IssueComment`

|               | GitHub | GitLab | Pagure |
| ------------- | :----: | :----: | :----: |

> | `body`-setter |   ✔    |   ✔    |   ✘    |

Reviewer: It is also getter, isn't it? It's not clear what the cross means. Does it mean that Pagure implementation does not have `body` at all?

What about two lines or `get`/`set` in the parentheses?
